### DISCUSSION TITLE
Why are dependencies not being pinned?

### DISCUSSION THREAD

**Discussion by thetayloredman** [renovatebot/renovate#19638] [Request Help] [answered]
On my repository [thetayloredman/Lexi](https://github.com/thetayloredman/Lexi), dependencies are not being pinned by Renovate and some updates aren't being PRd, such as `enmap@^5.9.5`. I assumed Renovate would make a PR to pin them, which is never happening, and I also expected Enmap to be updated (which might be suppressed because it's a range?)

**Comment by rarkins** [comment, marked as answer]
You need to configure rangeStrategy=pin

**Reply by thetayloredman** [reply]
Renovate made a PR to pin dependencies once - why isn't it doing it now? I thought pinning was default?

**Reply by rarkins** [reply]
It's no longer the default behavior for the config:base preset you're using
